### PR TITLE
Show image version on the main web page

### PR DESCRIPTION
**Pull request by okalachev**
To make identifying version of the software easier.

### CODE REVIEW

**Comment by okalachev** [comment]
@sfalexrog

File: clever/www/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1>CLEVER Drone Kit Tools</h1>
+<h1>CLEVER Drone Kit Tools <!-- $VERSION --></h1>
```
Comment (sfalexrog) [inline clever/www/index.html]: That would read as "Drone Kit Tools v0.19", as if this is only the tools version, not the image version itself. That is not technically incorrect, but would be somewhat misleading.

File: builder/image-ros.sh [hunk 2]
```diff
@@ -113,6 +113,9 @@ NPM_CONFIG_UNSAFE_PERM=true gitbook install
 gitbook build
 touch node_modules/CATKIN_IGNORE docs/CATKIN_IGNORE _book/CATKIN_IGNORE clever/www/CATKIN_IGNORE apps/CATKIN_IGNORE # ignore documentation files by catkin
 
+echo_stamp "Write down image version"
+sed -i "s|<!-- \$VERSION -->|$(</etc/clever_version)|g" clever/www/index.html
```
Comment (sfalexrog) [inline builder/image-ros.sh]: Not thrilled about changing a VCS-controlled file "on the fly". Don't we want `git status` to report no changes on a clean image?

Comment (okalachev) [reply builder/image-ros.sh]: Fair point.

**Comment by okalachev** [comment]
OK, keeping VCS file changed is not what we want, so I'll come up with another approach. Closing.

**Pull request closed without merging**